cam_width:  640
cam_height: 480
# cam_fx: 346.74048
# cam_fy: 349.12708
# cam_cx: 322.83899
# cam_cy: 234.54992

# cam_fx: 277.191356
# cam_fy: 277.191356
cam_fx: 554.382713
cam_fy: 554.382713
cam_cx: 320.5
cam_cy: 240.5


# realsense 
realsense_cam_fx: 462.266357421875
realsense_cam_fy: 462.266357421875
realsense_cam_cx: 320.0
realsense_cam_cy: 240.0

cam2body_R_realsense: [ 0.0, 0.0, 1.0,
                        -1.0, 0.0, 0.0,
                        0.0, -1.0, 0.0]
                    
cam2body_p_realsense: [0.12, 0.0, 0.0]


# cam2body_R_down: [ 0.0, -1.0,  0.0,
#                   -1.0,  0.0,  0.0,
#                     0.0,  0.0, -1.0]
                    
# cam2body_p_down: [-0.12, 0.0, -0.02]



# 原来的摄像头装反了,朝下的角度是1.0471975
# cam2body_R_front: [  0.0000000,  0.5000000,  0.8660254,
#                1.0000000, -0.0000000, -0.0000000,
#                0.0000000,  0.8660254, -0.5000000 ]

# cam2body_p_front: [0.037, 0.0, 0.004]

cam2body_R_down: [ 0.0, -1.0,  0.0,
                  -1.0,  0.0,  0.0,
                    0.0,  0.0, -1.0]
                    
cam2body_p_down: [-0.06 ,-0.01 ,-0.03]

# 使用朝下的向量乘上原始机体转相机
cam2body_R_front: [ 0 ,-0.865927 , 0.500171,
       -1      ,   0    ,     0,
        0, -0.500171 ,-0.865927]

cam2body_p_front: [0.08,0,-0.03]

